PRINCIPLES.md

```diff
@@ -158,3 +166,46 @@ _Principle: We help high-quality high-velocity cloud native open source projects
 We want to be able to say that CNCF is a net positive for big & small projects. Doing so requires more coordination with project leads. 
 
 Project needs may include test automation and CI, cloud resources to test on, clear documentation, per-project marketing & evangelism, roadmaps for interop, and advice from experts on governance and scalability. And we need to make sure project contributors see what value they are getting & are not afraid to ask for help!
+
+
+## Technical Leadership Principles
+
+### Purpose
+
+The Technical Oversight Committee has adopted these leadership principles to help guide the ecosystem in growth and maturity. These principles define the expectations of TOC members, project maintainers, TAG Chairs and Leads, chairs and leads of any group or committee, and other persons serving in any leadership role in the CNCF community.
```

Comment:
I like this. But, CNCF community is a little ambiguous and could be broadly or narrowly defined depending on the reader's knowledge. If the intent is to have this extend across the entire foundation's projects and subgroups or the TOC only, we probably need to be a little more precise with the language as I am not clear on the scope of CNCF community here.

It could be a one word update like "entire" if that's the scope.